packages/components/src/RadioGroup/RadioGroup.mdx

```diff
@@ -75,3 +76,55 @@ option will prevent a user from being able to select the option.
     );
   }}
 </Playground>
+
+## Description
+
+A `description` can be used to further describe the label of the `RadioOption`.
+
+<Playground>
+  {() => {
+    const [company, setCompany] = useState("apple");
+    return (
+      <RadioGroup onChange={setCompany} value={company}>
+        <RadioOption
+          value="apple"
+          label="Apple"
+          description="A delicious fruit that fends off doctors"
+        />
+        <RadioOption
+          value="amazon"
+          label="Amazon"
+          description="The worlds largest rainforest"
+        />
+        <RadioOption
+          value="google"
+          label="Google"
+          description="A search engine"
+        />
+      </RadioGroup>
+    );
+  }}
+</Playground>
+
+## Render RadioOption with Children
+
+If the `RadioGroup` is being used to select something other then text, you can
+render the `RadioOptions` with `children`.
+
+**Note:** When using `children`, `label` and `description` are not allowed.
```

Comment:
It might help to explain why they aren't allowed here. Also might be good to include a bit more explanation about how screen readers will use this label and description, and how each should be written (short and longer, respectively).